components/squeezelite/equalizer.c

```diff
@@ -148,18 +148,28 @@ void equalizer_set_volume(unsigned left, unsigned right) {
  */
 void equalizer_set_gain(int8_t *gain) {
 #if BYTES_PER_FRAME == 4
+    static uint8_t last_gain[EQ_BANDS] = { };
```

Comment:
Do you need that static (it consumes a bit of DRAM => expensive). Couldn't we use the memorized equalizer.gain, with a cast ?